### PR TITLE
navbar-and-some-style

### PR DESCRIPTION
**Pull request by Manu7864z**
- created Navbar Compenent
- added states to toggle and show on which site the user is 
- added reac-icons, fill and outline to toggle
- added some styled to the calendar and Navbar

### CODE REVIEW

File: components/NavBar/index.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import Link from "next/link";
+import styled from "styled-components";
+import { AiFillHome } from "react-icons/ai";
+import { useState } from "react";
+import { AiOutlineHome } from "react-icons/ai";
+import { RiCalendarTodoFill } from "react-icons/ri";
+import { RiCalendarTodoLine } from "react-icons/ri";
+import { BsFilePersonFill } from "react-icons/bs";
+import { BsFilePerson } from "react-icons/bs";
```
Comment (Kheirah) [inline components/NavBar/index.js:9]:
```suggestion
import { useState } from "react";
import { AiFillHome, AiOutlineHome } from "react-icons/ai";
import { RiCalendarTodoFill, RiCalendarTodoLine  } from "react-icons/ri";
import { BsFilePersonFill, BsFilePerson  } from "react-icons/bs";
```
You can import several things as the same time as long as they come from the same place =)

File: components/NavBar/index.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+export default function Navbar() {
+  const [home, setHome] = useState(false);
+  const [newTodo, setNewTodo] = useState(false);
+  const [about, setAbout] = useState(false);
```
Comment (Kheirah) [inline components/NavBar/index.js:14]: Not the most elegant way to solve navigation 😆 
A better solution would be to not use states at all. Instead, you'd link to different routes like `/activitiesForm`, the homepage and an about page. Within those routes you would check the current url and pass down its value to the `NavBar` component which in turn knows which icon to display depending on the value it receives.

**Review by Kheirah — CHANGES_REQUESTED** [review]
See if you can find a better way to display the filled icons depending on the route you're in. I left some comments to get you started in the right direction.

File: components/NavBar/index.js (hunk 2 above)
Comment (Manu7864z) [reply components/NavBar/index.js:14]: i knew this would be an issue but i was so proud that it works as desired 🤣😢
I'try to change it.

**Pull request merged**